components/guidance_feedback/guidance-feedback.react.js

```diff
@@ -0,0 +1,146 @@
+    
+    constructor(props){
+        super(props);
+        
+        // Set properties
+        this.textarea = null;
+        this.objGTM = {};
+
+        // Bind methods to the class component construtor
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.buildGTMObj = this.buildGTMObj.bind(this);
+        this.updateStateBtnNo = this.updateStateBtnNo.bind(this);
+        this.updateStateBtnYes = this.updateStateBtnYes.bind(this);
+        this.pushInDataLayer = this.pushInDataLayer.bind(this);
+
+        this.setTextareaRef = element => {
+            this.textarea = element;
+        };
+
+        // Define default state props
+        this.state = {
+            btnVisible: true,
+            formVisible: false,
+            label:'',
+            fieldId:'',
+            message: 'Did you find the guidance you needed?',
+        }
+    }    
+
+    buildGTMObj(obj) {
+        // Focus the text input using the raw DOM API
+        if (this.textarea) {
+            obj.event = 'Feedback';
+            obj.eventCategory= 'Research guides feedback';
+            obj.eventLabel = this.textarea.value === '' ? 'No comment made' : this.textarea.value ;
+            obj.eventAction = this.textarea.id === 'field-no'? 'No' : 'Yes';
+            return obj;
+        }
+    }
+
+    pushInDataLayer(obj) {
+        let wd = window.dataLayer || [];
+        (!!obj || typeof obj === 'object') ? wd.push(obj) : '';
+    
+        return obj;
+    }
+
+    updateStateBtnYes(){
+        this.setState(state => ({
+            btnVisible: !state.btnVisible,
+            formVisible: !state.formVisible,
+            label:'Any comments on your experience?',
+            fieldId:'field-yes'
+        }));
+    }
+    updateStateBtnNo(){
+        this.setState(state => ({
+            btnVisible: !state.btnVisible,
+            formVisible: !state.formVisible,
+            label:'What did you expect to find?',
+            fieldId:'field-no'
+        }));
+    }
+
+    handleSubmit(event) {
+        event.preventDefault();
+
+        this.setState(state => ({
+            btnVisible: false,
+            formVisible: false,
+            label:state.label,
+            fieldId:state.fieldId,
+            message: 'Thank you for your feedback.',
+        }));
+
+        // Build the GTM Object
+        this.buildGTMObj(this.objGTM);
+        // Push the GTM Object in dataLayer
+        this.pushInDataLayer(this.objGTM);
+
+        return true;
+    }
+    
+    render() {
+        const btnVisible    = this.state.btnVisible,
+              formVisible   = this.state.formVisible,
+              fieldLabel    = this.state.label,
+              fieldId       = this.state.fieldId;
+
+        return (
+            <form action="" id="guidance_feedback" className="react_component_gf">
+                <h2>Feedback</h2>
+                <h3 id="aria">{ this.state.message}</h3>
+                <Button 
```

Comment:
This should have an `aria-label="No"`
Please refer to link [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)

Reply:
Resolved